Hymns/Display/DisplayHymnBottomBar.swift

```diff
@@ -45,6 +46,12 @@ struct DisplayHymnBottomBar: View {
                     }
                 }
             }
+            if tagOn {
+                VStack {
+                    Divider()
+                    TagSheetView(viewModel: TagSheetViewModel(hymnToDisplay: self.viewModel.identifier), tagOn: self.$tagOn).eraseToAnyView()
+                }
+            }
```

Comment:
Can we make this a bottomSheet instead?

Reply:
So, I had done that previously in some earlier commits. But it was not resizing correctly whenever the tags would go to empty or populate as is needed from the figma specs. When the view would update and redraw it would be buggy with the tags sometimes not redrawing onto the view. This way here is super smooth. It redraws to the correct size everytime and is still coming up at the bottom. 

What is the benefit of making it a bottomSheet?

Reply:
Can you elaborate what you mean? Or show me a video? I just tried it and it seems okay. The only problem is that I don't know how to make the sheet non-full screen

I think the benefit is
1) We don't need to do the opacity and stuff ourselves
2) There's a nice little animation that also minimizes the background a little bit
3) It's generally better not to reinvent the wheel if we can use something built-in

Reply:
So the way I had it automatically redraws when a tag is removed or added. See the two images below and how the size changes to match what is necessary. What I mean't though about not redrawing and stuff was when I was using a custom sized geometry reader bottom view so as to make it not take up the whole screen. I think the whole solution I was trying to develop is with the point that we don't want to use a whole sheet that covers everything. 

I think the main thing is that everything I have been doing here is basically with the idea that we need a solution that is not full screen like built in sheet. And unfortunately, sheet at this point just does not support anything but being full size. 

1. I agree it is annoying to have to do opacity ourselves but unfortunately that is going to be necessary if we don't want it to take the whole screen with a big sheet. 

To me it's worth the custom implementation in order to gain the result of it not being a full sheet and matching the figma sheet better. But I do wish the backgound animation and blur can be improved. Idk what do you think? Go with the full sheet?

![Simulator Screen Shot - iPhone 8 - 2020-06-13 at 09 44 42](https://user-images.githubusercontent.com/15247726/84571714-a2e80d00-ad5a-11ea-8b71-e62dfa498a4e.png)

![Simulator Screen Shot - iPhone 8 - 2020-06-13 at 09 44 48](https://user-images.githubusercontent.com/15247726/84571711-a24f7680-ad5a-11ea-8b48-56edb1af76ab.png)

Reply:
We can add some blur as well as opacity to make it look more like a sheet though
![Simulator Screen Shot - iPhone 8 - 2020-06-13 at 10 11 57](https://user-images.githubusercontent.com/15247726/84572504-ff016000-ad5f-11ea-95b9-08c2b1d6147b.png)
. See below